xtraplatform-features-sql/src/main/java/de/ii/xtraplatform/features/sql/app/DecoderFactorySqlExpression.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 interactive instruments GmbH
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package de.ii.xtraplatform.features.sql.app;
+
+import com.github.azahnen.dagger.annotations.AutoBind;
+import de.ii.xtraplatform.base.domain.util.Tuple;
+import de.ii.xtraplatform.features.domain.Decoder;
+import de.ii.xtraplatform.features.domain.DecoderFactory;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.core.MediaType;
+
+@Singleton
+@AutoBind
+public class DecoderFactorySqlExpression implements DecoderFactory {
+
+  public static final MediaType MEDIA_TYPE = MediaType.valueOf("application/sql+expression");
```

Comment:
This is only an internal identifier, so it is sufficient if it is accepted by the `MediaType` class. And I think I did not want to use `application/sql` since it is not plain SQL but includes the  `{sql=` wrapper and since that might be used for something else in the future.

Reply:
OK, then we should use something like "application/vnd.ldproxy.sql-expression" (that is, a media type that we could register).

Reply:
Done.